packages/eslint-config-terra/UpgradeGuide.md

```diff
@@ -0,0 +1,20 @@
+# Eslint Config Terra Upgrade Guide
```

Comment:
mention we're dropping node 8 support?

Reply:
Added https://github.com/cerner/terra-toolkit/pull/402/commits/fccdced0fe1481137dc732cec09bceeb8560830f